modules/op-pac-concepts.adoc

```diff
@@ -0,0 +1,29 @@
+// This module is included in the following assemblies:
+// * pac/using-pipelines-as-code-repos.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="pac-concepts_{context}"]
+= {pac} concepts
+
+{pac} interacts with your Git repository provider. To use {pac}, you must first configure this integration.
+
+You can use {pac} with any number of repositories within your Git repository provider. For each repository, you must create a `Repository` custom resource (CR) within an {OCP} namespace that you control. This CR contains information that {pac} can use to access this repository.
+
+{pac} starts the pipeline runs for a repository within the namespace in which the `Repository` CR for this repository is located.
+
+Inside your Git repository, you must create a `.tekton` directory and store pipeline run definitions as YAML files in this directory. Each pipeline run that you define must include annotations that determine the events that must trigger this pipeline run.
+
+The definition can reference other YAML files in the same directory. For example, you can define a pipeline in a separate YAML file and refer to this pipeline in the pipeline run file. You can also use annotations in the pipeline run definition to reference task and pipeline resources from Tekton Hub, from HTTP locations, and from other paths in your directory. You can use special variables in the definition to reference execution context, such as the name of the branch.
+
+When a matching event happens, {pac} creates a `PipelineRun` CR based on the definition that you provided in the repository. Creating a `PipelineRun` CR triggers execution of the pipeline that is defined in the pipeline run.
+
+{pac} uses the _{pac} resolver_ to create the `PipelineRun` CR based on your definition. The {pac} resolver retrieves all of the resources that you referenced by using annotations and adds them to the `PipelineRun` CR. If the {pac} resolver fails to retrieve any referenced resource, {pac} logs an error and does not create a pipeline run.
+
+{pac} also replaces dynamic variables in your pipeline run definition with their values. 
+
+[IMPORTANT]
+====
+By default, when creating a `PipelineRun` CR for a pull request or push event, {pac} uses the definitions from the `.tekton` directory of the source branch of the pull request or push event. If the pull request or push event modifies the `.tekton` directory, {pac} uses the modified version.
+
+You can define the `default_branch` parameter in the `Repository` CR to change this behaviour. If you defined the `default_branch` parameter, {pac} always uses definitions from the `.tekton` directory of the specified branch for any pipeline runs for the specified repository.
```

Comment:
what do you mean by the specified branch, you probably want to explain it's the default branch as configured in github/gitlab etc..

Reply:
I thought you specified the branch in the `default_branch` parameter. So instead it uses the default branch configured in the Git provider, such as `main` ?

Reply:
I have checked toe docs and yes, I misunderstood this. Rewritten and pushed.